webapps/density-custom-chart/style.css

```
@@ -0,0 +1,100 @@
+@import '/static/public/styles/1.0.0/dku-styles.css';
+
+#spinner {
+    display: none;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    z-index: 3000;
+    width: 150px;
+    height: 150px;
+    margin: -75px 0 0 -75px;
+    border: 16px solid #f3f3f3;
+    border-radius: 50%;
+    border-top: 16px solid #3498db;
+    -webkit-animation: spin 2s linear infinite;
+    animation: spin 2s linear infinite;
+}
+
+@-webkit-keyframes spin {
+    0% { -webkit-transform: rotate(0deg); }
+    100% { -webkit-transform: rotate(360deg); }
+}
+
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
+
+.user-message-style {
+    z-index: 1500;
+    background-color: #28a9dd;
+    position: absolute;
+    top: 0px;
+    left: -1px;
+    margin-top: 10px;
+    width: 100%;
+    text-align: center;
+    padding: 10px;
+    font-family: 'SourceSansPro';
+    font-weight: 500;
+    font-size: 14px;
+    color: #ffffff;
+    box-sizing: border-box;
+}
+
+.error-message-style {
+    z-index: 1500;
+    background-color: #28a9dd;
```

You'd better use Dataiku vars
```suggestion
    background-color: var(--error-red);
```

---

This one seems to be blue, skipping it

---

Where are those vars ?

---

You can find them in the following file: http://localhost:11200/static/public/styles/1.0.0/variables.css